R/github-actions.R

```diff
@@ -4,51 +4,44 @@
 #' GitHub using [GitHub Actions](https://github.com/features/actions). These functions
 #' - Add the necessary configuration files and place them in `.Rbuildignore`.
 #' - Provide the markdown to insert a badge into your README
-#' @name actions
+#' @name github_actions
 NULL
 
 #' @section `use_github_actions()`:
-#' Adds a basic ‘R.yaml’ file to the `.github/workflows` directory of a
+#' Adds a basic ‘R-CMD-check.yaml’ file to the `.github/workflows` directory of a
 #'  package. This is a configuration file for the [GitHub
 #'  Actions](https://github.com/features/actions) service.
-#' @rdname actions
+#' @rdname github_actions
 #' @export
 use_github_actions <- function() {
   check_uses_github()
 
-  new <- use_action_check_release()
-
-  if (!new) {
-    return(invisible(new))
-  }
-  use_github_actions_badge("R-CMD-check")
-
-  invisible(new)
```

Comment:
Test coverage revealed the bottom lines would never get run, as `use_action_check_release()` already adds the badge, so we don't need to do it twice.